### PR TITLE
JI:773108 P2022-2103 [Visual] Invisible part of the text and moving content when sliding sideways

### PR DESCRIPTION
**Pull request by krysia1**
…ection on small devices

### CODE REVIEW

File: apps/site/src/app/pages/not-found/index.tsx [hunk 1]
```diff
@@ -8,14 +7,12 @@ const NotFoundPage: React.FC = () => {
...
-    <>
-      <Styled.MediumImageWrapper>
-        <CardMedia component='img' alt={HttpStatus.NotFound} src={NotFoundImage} />
-      </Styled.MediumImageWrapper>
-
-      <h1>{t("notFoundPage.title")}</h1>
-    </>
+    <Box>
+      <Typography variant='h1' align='center'>
+        404
```
Comment (Reden4005) [inline apps/site/src/app/pages/not-found/index.tsx:12]: Why you removed image?

Comment (Reden4005) [inline apps/site/src/app/pages/not-found/index.tsx:12]: Why you removed image?

Comment (krysia1) [reply apps/site/src/app/pages/not-found/index.tsx:12]: the image was black and it was hardly seen in dark mode

**Review by aniasgit — COMMENTED** [review]
LGTM

**Comment by piotrrawski** [comment]
LGTM

**Review by pawelwojdas — COMMENTED** [review]
LGTM

File: apps/site/src/app/pages/not-found/index.tsx (hunk 1 above)
Comment (kacperChlon) [inline apps/site/src/app/pages/not-found/index.tsx:12]: please add this (or reuse if exists) to shared types under HttpStatusCode { 
NotFound = 404
}

**Review by kacperChlon — APPROVED** [review]

File: apps/site/src/app/pages/not-found/index.tsx (hunk 1 above)
Comment (krysia1) [reply apps/site/src/app/pages/not-found/index.tsx]: kk

**Review by pawelwojdas — APPROVED** [review]

**Pull request merged**